src/implementations/astroport/helpers.rs

```diff
@@ -71,3 +76,113 @@ pub(crate) fn astro_asset_info_to_cw_asset_info(asset_info: &AstroAssetInfo) ->
         } => AssetInfo::cw20(contract_addr.clone()),
     }
 }
+
+/// ## Description
+/// Compute the current pool amplification coefficient (AMP).
+/// ## Params
+/// * **config** is an object of type [`Config`].
+///
+/// * **env** is an object of type [`Env`].
```

Comment:
could not find this function in the astro code on the commit you linked in the other comment @georgemc98 

https://github.com/astroport-fi/astroport-core/tree/c216ecd4f350113316be44d06a95569f451ac681/contracts/pair_stable/src

Reply:
Here it is: https://github.com/astroport-fi/astroport-core/blob/c216ecd4f350113316be44d06a95569f451ac681/contracts/pair_stable/src/contract.rs#L1486

Reply:
yeah i found it and pushed a commit with the update